### ISSUE TITLE
audio slideshow error reported by Wave when headings are empty

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#938]
Testing with wave. When the audio slideshow page has empty headings on each page, as is sometimes desirable as it is in our demonstrator https://xot.xerte.org.uk/play.php?template_id=116#audioslideshow then wave reports a missing headings error. We should probably check if the heading is empty and if so hide the relevant h3 tag too so that an error isn't reported.
I suspect the same issue might apply to other pages with nested pages where it's sometimes desirable to leave the headings empty.
